api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Is this generated automatically?

Reply:
I assume it added by the `goimports` tool